### ISSUE TITLE
Empty records `{}` and lists `[]` aren't rendered well in the output

### ISSUE THREAD

**Issue by webbedspace** [nushell/nushell#6765]
### Describe the bug

As an experiment, I tried seeing what empty records, empty lists and empty tables look like when output. I was disappointed with the first two, which are simply displayed as either empty lines, or nothing at all.

### How to reproduce

```
〉{}

〉[]
〉[[]; []]
╭───┬───────────────────╮
│ 0 │ {record 0 fields} │
╰───┴───────────────────╯
```

### Expected behavior

I expect empty records to be rendered as *something*, like maybe `{record 0 fields}` on its own line, and empty lists to be represented as `[]` or `[list 0 items]`. Showing nothing is just frustrating.

### Configuration

| key                | value                                           |
| ------------------ | ----------------------------------------------- |
| version            | 0.69.1                                          |
| branch             |                                                 |
| commit_hash        | 8a5abc7afc28a36908a32f1852127e2a5170b461        |
| build_os           | windows-x86_64                                  |
| build_target       | x86_64-pc-windows-msvc                          |
| rust_version       | rustc 1.64.0 (a55dd71d5 2022-09-19)             |
| rust_channel       | stable-x86_64-pc-windows-msvc                   |
| cargo_version      | cargo 1.64.0 (387270bc7 2022-09-16)             |
| pkg_version        | 0.69.1                                          |
| build_time         | 2022-09-28 02:58:08 +00:00                      |
| build_rust_channel | release                                         |
| features           | database, dataframe, default, trash, which, zip |
| installed_plugins  |                                                 |

**Comment by fdncred** [comment]
Hints for anyone looking to work on this, my guess is that you could start with looking at:
- `[]` in `parse_table_expression`
- `{}` in `parse_record`
in parser.rs. But that's just a guess and could be wrong.

**Comment by rgwood** [comment]
Implemented in https://github.com/nushell/nushell/pull/7925

@webbedspace you may want to take a look at that PR. We're not 100% sure we'll keep it for v0.77; there's been some discussion about how it works with `each` [here](https://github.com/nushell/nushell/pull/8014#issuecomment-1440733867).

**Comment by KAAtheWiseGit** [comment]
This was fixed at some point:

```Nushell
~> {}
╭──────────────╮
│ empty record │
╰──────────────╯
~> []
╭────────────╮
│ empty list │
╰────────────╯
```

**Comment by sholderbach** [comment]
Thanks for the heads-up @KAAtheWiseGit 

Closed by #7925
